src/core/algorithms/ind/ind.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace model {
+
+using AttributeIndex = unsigned int;
+
+class IND {
+public:
+    class ColumnCombination {
```

Comment:
Not sure if it should be a nested class defined in `IND`, it seems like a class that may be useful in, for example, some FD mining algorithm, it doesn't look specific for inclusion dependencies.
We also have (unused) `src/core/model/table/column_combination.h`, that should be deleted. Maybe it's a good idea to move this definition there, but somehow indicate in its name that it uses vector to store set of attributes